tests/apply-shim.html

```diff
@@ -138,6 +138,23 @@
     </style>
     </custom-style>
   </template>
+  <template id="important">
+    <style>
+    :host {
+      --mixin-important: {
+        border: 2px solid black !important;
+        background-color: white;
+      };
+      --mixin: {
+        border: 1px dotted orange;
```

Comment:
Please also add a `background-color: red;` here

Reply:
Could you also add a case where both properties have `!important` and assert the last one is actually used?